node/src/reactor/participating.rs

```diff
@@ -482,11 +482,14 @@ impl reactor::Reactor for Reactor {
         ));
 
         let execution_pre_state = match maybe_latest_block_header {
+            // if there is a latest block header and it's later than the block that was highest
```

Comment:
```suggestion
            // if there is a latest block header and it's later or equal than the block that was highest
```

Reply:
The "or equal" wouldn't be true, though - because the "greater than or equal" compares the height of the latest block with the height of the _next_ block _after_ the highest block at the time of initialization. so, even if these are equal, the latest block header is _still_ later than the other one.

Reply:
Of course. I forgot about how the `ExecutionPreState` is constructed.